### PR TITLE
Feature/desktop version

### PR DESCRIPTION
**Pull request by melkamu372**
### features added to the new branch feature/desktop-version

- add desktop version style to the whole project
- add desktop version style for intro
- add work section desktop version styles
- add contact form desktop version
- fix style bug

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -37,11 +37,15 @@ label {
...
-  max-width: 30.5rem;
+  width: 100%;
   top: 1rem;
   background-color: #ffff;
 }
 
+.nav-menu {
+  display: none;
+}
+
```
Comment (ademibowale) [inline style.css:48]: - [ ] great job and kudo to you, as you have done a great job on this milestone I can say your project is great and look exactly like Figma template but according to this project requirement, we are to check if this mobile design still looks like expected, but as you can see the screenshot below .showing the mobile menu icon not align with the name Navigation logo compare with the original design onmobile, kindly fix the mobile menu icon or The hamburger button should be aligned with the navigation name logo, to make your website look professional.
- [ ] Your design
Screenshot 2023-07-20 212844

- [ ] figma design
Screenshot 2023-07-20 212738

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @melkamu372  , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/contact/contact-form-desktop.svg
Comment (AbelG101) [inline assets/contact/contact-form-desktop.svg:1]: - [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Key Features & Future Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊
  image
  image

**Review by AbelG101 — APPROVED** [review]
# Hi @melkamu372 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI is looking good ✔
- Proper branch name ✔
- Correct use of GitHub Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**